tests/unittests/odm_tests.py

```
@@ -109,3 +109,85 @@ def test_get_indexed_values(self):
+        data = {
+            "field": "value",
+            "nested": {
+                "_cls": "fiftyone.core.odm.embedded_document.DynamicEmbeddedDocument",
+                "data": "preserved",
+            },
+        }
+
+        result = foou.sanitize_unknown_embedded_docs(data)
+
+        # Should not raise an error and should preserve structure
+        self.assertEqual(result["field"], "value")
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)